leetcode/generate_md.py

```diff
@@ -5,7 +5,10 @@
 head = head.replace('\n', '')
```

Comment:
VERIFICATION RESULT : 
 Hello! 
 I am here to help you with making your pr title looks better. Now it is: 
 fixed bug w\ more than two words in head 
 Mistakes: 
 Where have you found such homework(fixed)? It is not from ['LEETCODE', 'GENERATOR', 'TRIANGLE', 'HEXNUMBER', 'REQUESTS'].
There is only one group(bug), how could you make mistake here? So, it is definetly not 1013.
Wait...What did you do(w\)? It is not from ['Added', 'Deleted'].